lib/solidus_paypal_commerce_platform/engine.rb

```diff
@@ -23,6 +23,14 @@ class Engine < Rails::Engine
       app.config.spree.payment_setup_wizards << "SolidusPaypalCommercePlatform::Wizard"
     end
 
+    def self.activate
+      Dir.glob(File.join(File.dirname(__FILE__), '../../app/**/*_decorator*.rb')) do |c|
+        Rails.configuration.cache_classes ? require(c) : load(c)
+      end
+    end
+
+    config.to_prepare(&method(:activate).to_proc)
+    
```

Comment:
This is not necessary if we move all decorators to `app/decorators`, which is the default path for `solidus_support`.

Reply:
Oh great, `solidus_support` really does a lot under the hood. Thanks for pointing this out!